index.src.html

```diff
@@ -1,9 +1,9 @@
 <h1>Reporting API</h1>
 <pre class="metadata">
-Status: CG-DRAFT
-ED: https://wicg.github.io/reporting/
+Status: w3c/CG-DRAFT
```

Comment:
There's no "status of this document" section in the preview.  I think this needs to be `ED` — @igrigorik, "Editor's Draft" is the right status now that this is in WebPerf instead of WICG, right?

Reply:
ED seems correct for a spec that's not ready for FPWD yet; that's the current state of the other two specs I can find which have migrated from WICG.